risc0/zkvm/src/receipt/succinct.rs

```diff
@@ -213,6 +229,52 @@ where
     }
 }
 
+/// Constructs the set of allowed control IDs, given a maximum cycle count as a po2.
+pub(crate) fn allowed_control_ids(
+    hash_name: impl AsRef<str> + 'static,
+    po2_max: usize,
+) -> anyhow::Result<impl Iterator<Item = Digest>> {
+    // Recursion programs (ZKRs) that are to be included in the allowed set.
+    // NOTE: Although the rv32im circuit has control IDs down to po2 13, lift predicates are only
+    // generated for po2 14 and above, hence the magic 14 below.
+    let allowed_zkr_names: BTreeSet<String> = ["join.zkr", "resolve.zkr", "identity.zkr"]
+        .map(str::to_string)
+        .into_iter()
+        .chain((14..=po2_max).map(|i| format!("lift_{i}.zkr")))
```

Comment:
Suggest we use a `const MIN_RECURSION_PO2: usize = 14`

Reply:
Addressed in f44d4ffc7dd647dd2f8bb8c7ed22c419455f58db